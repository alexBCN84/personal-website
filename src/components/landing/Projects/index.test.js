import React from 'react';
import { render } from '@testing-library/react';
import { Projects } from './index';

describe('Projects component', () => {
    it('renders without crashing', () => {
        const { container } = render(<Projects />)
        expect(container.firstChild).toMatchSnapshot();
    });
});