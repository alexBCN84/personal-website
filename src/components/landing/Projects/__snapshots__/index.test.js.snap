// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects component renders without crashing 1`] = `
<div
  id="projects"
  style="background-color: rgb(212, 172, 13); width: 100vw; height: 100vh; padding-top: 60px;"
>
  <div>
    <h1>
      Projects
    </h1>
  </div>
</div>
`;
